ocaml/xapi/storage_migrate.ml

```diff
@@ -300,7 +300,9 @@ let copy' ~task ~dbg ~sr ~vdi ~url ~dest ~dest_vdi =
 				);
 			)
 			(fun () -> 
-				Remote.DP.destroy ~dbg ~dp:remote_dp ~allow_leak:false);
+				Remote.DP.destroy ~dbg ~dp:remote_dp ~allow_leak:false;
+				State.remove id State.active_copy
```

Comment:
So it is symmetrical - good - and this commit fixes the oversight to reverse the `State.add_to_active` in the `finally` section.